setup.py

```diff
@@ -5,10 +5,14 @@
 from distutils.sysconfig import get_python_lib
 from torch.utils.cpp_extension import BuildExtension, CUDAExtension
 
-os.environ["CC"] = "g++"
-os.environ["CXX"] = "g++"
+if "CC" not in os.environ:
+    os.environ["CC"] = "g++"
+if "CXX" not in os.environ:
+    os.environ["CXX"] = "g++"
 AUTOAWQ_KERNELS_VERSION = "0.0.7"
 PYPI_BUILD = os.getenv("PYPI_BUILD", "0") == "1"
+COMPUTE_CAPABILITIES = os.getenv("COMPUTE_CAPABILITIES", "75,80,86,87,89,90")
```

Comment:
I've also added a way to to define more computing architectures without editing the file.
PS: I merged #17 to add support for Jetson Orin devices (`87`)